src/app/ticker/ticker.component.scss

```diff
@@ -34,10 +37,6 @@
   width: 100%;
   position: relative;
   overflow: hidden;
-  top: 1px;
```

Comment:
@alihejazi removing `top` completely on a `relative` element seems like a risk for regressions in some browsers. I think the original design also had the smaller ticker pieces vertically aligned in the middle of the primary value.